python/hail/plot/plots.py

```diff
@@ -54,8 +80,21 @@ def scatter(x, y, label=None, title=None, xlabel=None, ylabel=None, size=4):
     -------
     :class:`bokeh.plotting.figure.Figure`
     """
+    if isinstance(x, Expression):
```

Comment:
we want to execute both x and y at the same time here as well. 

Should this support x being an Expression and y being a list? Maybe yes, maybe no. I'll leave that up to you, but you only get 1 aggregation! And if you decide not to support it, then add a check and a nice error.